### PR TITLE
Gh/ts/patch 020 service rtc codec mime type

### PR DESCRIPTION
**Pull request by garyhuntddn**

### CODE REVIEW

**Comment by saghul** [comment]
Jenkins please test this please.

File: JitsiConference.js [hunk 1]
```diff
@@ -1612,6 +1612,14 @@ JitsiConference.prototype.getLastN = function() {
     return this.receiveVideoController.getLastN();
 };
 
+/**
+ * Obtains the forwarded sources list in this conference.
+ * @return {Array<string>|null}
+ */
+JitsiConference.prototype.getForwardedSources = function() {
```
Comment (saghul) [inline JitsiConference.js:1619]: Unrelated change?

File: modules/RTC/BridgeChannel.js [hunk 2]
```diff
@@ -359,11 +359,25 @@ export default class BridgeChannel {
                 break;
             }
             case 'LastNEndpointsChangeEvent': {
-                // The new/latest list of last-n endpoint IDs (i.e. endpoints for which the bridge is sending video).
-                const lastNEndpoints = obj.lastNEndpoints;
+                if (!FeatureFlags.isSourceNameSignalingEnabled()) {
```
Comment (saghul) [inline modules/RTC/BridgeChannel.js:362]: Where does this come from? Bad merge perhaps?

**Review by saghul — CHANGES_REQUESTED** [review]
I think there was some weird merge  going on here...

File: modules/RTC/BridgeChannel.js (hunk 2 above)
Comment (garyhuntddn) [reply modules/RTC/BridgeChannel.js:362]: Yeah - that'll be a bad merge - I was plagued with them yesterday with these changes that are affecting multiple files - I think going forward we should do them in multiple steps
